MdeModulePkg/Core/Dxe/Misc/MemoryProtection.c

```diff
@@ -1179,20 +1186,16 @@ MemoryProtectionExitBootServicesCallback (
   }
 }
 
-/**
-  Disable NULL pointer detection after EndOfDxe. This is a workaround resort in
-  order to skip unfixable NULL pointer access issues detected in OptionROM or
-  boot loaders.
+// MU_CHANGE [START]: Move disable NULL detection to separate routine so it can be called
+//                    outside of the event context.
 
-  @param[in]  Event     The Event this notify function registered to.
-  @param[in]  Context   Pointer to the context data registered to the Event.
+/**
+  Disable NULL pointer detection.
 **/
-// MU_CHANGE START Update naming due to addition of ability to disable at readytoboot
 VOID
 EFIAPI

```

Comment:
If `DisableNullProtection()` is only being called internally now, you can drop `EFIAPI` from the signature.

Reply:
Updated